homework2/task04/tests/test_cach.py

```diff
@@ -0,0 +1,43 @@
+from typing import Callable, Tuple
+
+import pytest
+from project4.task4 import cache
+
+
+def function(a: int, b: int) -> int:
+    """Return result of math operation on arguments."""
+    return (a ** b) ** 2
+
+
+some = [1, 2]
+
+
+def test_func_positive():
+    cache_function = cache(function)
+    assert cache_function(*some) is cache_function(*some)
+
+
+def test_func_negative():
+    cache_function = cache(function)
+    assert cache_function(1, 2) is not cache_function(2, 1)
+
+
+def test_sum():
+    def summa(*args) -> int:
```

Comment:
You don't need to write your own `summ`, just cache `sum`

Reply:
Ok, fixed